api/app/src/command/medical/document/document-query.ts

```diff
@@ -17,6 +18,22 @@ import {
 import { appendDocQueryProgress, SetDocQueryProgress } from "../patient/append-doc-query-progress";
 import { getPatientOrFail } from "../patient/get-patient";
 
+export function isProgressEqual(a?: Progress, b?: Progress): boolean {
+  return (
+    a?.errors === b?.errors &&
+    a?.status === b?.status &&
+    a?.successful === b?.successful &&
+    a?.total === b?.total
+  );
+}
+
+export function isDocumentQueryProgressEqual(
+  a?: DocumentQueryProgress,
+  b?: DocumentQueryProgress
+): boolean {
+  return isProgressEqual(a?.convert, b?.convert) && isProgressEqual(a?.download, b?.download);
```

Comment:
I think its NABD, but in theory we don't need to check `download`, only `convert`

Reply:
"In theory" haha - will leave this in just in case we see weird behavior with the `download` db update in the future.